slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -292,6 +292,12 @@ class SLAMServiceImpl final : public SLAMService::Service {
     std::string latest_jpeg_map_without_marker;
 
     std::string latest_pointcloud_map;
+
+    // Enable Localization mode to utilize a cached version of the pointcloud
+    // map, to skip unecessary encoding steps
+    std::atomic<bool> localization_map_ready{false};
+    std::string localization_pointcloud_map;
```

Comment:
nit: these should be declared above the mutexes, so it is clear they are not guarded by the mutexes.

Reply:
will do, figured since I was treating the atomic bool as a lock that it should live down there